package.json

```diff
@@ -25,17 +25,34 @@
   "unpkg": "./dist/fuzzy-search.umd.js",
   "types": "./dist/fuzzy-search.d.ts",
   "exports": {
-    "require": "./dist/fuzzy-search.cjs",
-    "types": "./dist/fuzzy-search.d.ts",
-    "default": "./dist/fuzzy-search.modern.js"
+    ".": {
+      "module-sync": {
```

Comment:
How is "module-sync" used, what uses it? I have not seen this before.

Reply:
It's a new exports condition that allows for synchronously requiring an ESM file within CJS files.

[**docs**](https://nodejs.org/en/blog/release/v22.10.0#new-module-sync-exports-condition)